src/main/java/uk/gov/hmcts/darts/dailylist/mapper/DailyListMapper.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.darts.dailylist.mapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.experimental.UtilityClass;
+import uk.gov.hmcts.darts.common.entity.Courthouse;
+import uk.gov.hmcts.darts.common.entity.DailyListEntity;
+import uk.gov.hmcts.darts.dailylist.enums.JobStatusType;
+import uk.gov.hmcts.darts.dailylist.exception.DailyListException;
+import uk.gov.hmcts.darts.dailylist.model.DailyListPostRequest;
+import uk.gov.hmcts.darts.dailylist.model.DocumentID;
+
+@UtilityClass
+public class DailyListMapper {
+
+    public DailyListEntity mapToDailyListEntity(DailyListPostRequest postRequest, Courthouse courthouse) {
+
+        DailyListEntity dailyListEntity = new DailyListEntity();
+        dailyListEntity.setVersion((short) 1);
+        mapToExistingDailyListEntity(postRequest, courthouse, dailyListEntity);
+        return dailyListEntity;
+    }
+
+    public void mapToExistingDailyListEntity(DailyListPostRequest postRequest, Courthouse courthouse, DailyListEntity dailyListEntity) {
+        uk.gov.hmcts.darts.dailylist.model.DailyList dailyList = postRequest.getDailyList();
+        DocumentID documentId = dailyList.getDocumentId();
+        dailyListEntity.setCourthouse(courthouse);
+        dailyListEntity.setUniqueId(documentId.getUniqueId());
+        dailyListEntity.setStatus(String.valueOf(JobStatusType.NEW));
+        dailyListEntity.setTimestamp(documentId.getTimeStamp());
+        dailyListEntity.setStartDate(dailyList.getListHeader().getStartDate());
+        dailyListEntity.setEndDate(dailyList.getListHeader().getEndDate());
+        dailyListEntity.setSource(postRequest.getSourceSystem());
+
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Can we use the `ObjectMapper` that Spring Boot uses it, or do we need to customise it?